ets-demo/etstool.py

```diff
@@ -127,6 +130,23 @@
 
 doc_dependencies = {}
 
+full_app_dependencies = [
+    'eam',
+    'traitsui',
+    'etsdemo',
+    'envisage',
+    "apptools",
+    "chaco",
+    "h5py",
+    "numpy",
+    "pandas",
+    "pygments",
+    "pytables",
+    "pyqt5"
```

Comment:
We need to have a single specified toolkit.  `pyqt5` may ultimately cause problems do to `QtWebKit` issues.  See  https://github.com/enthought/enable/pull/494#issuecomment-745473789
We may want `pyside2` instead?   I'm not sure, but we will need to settle on a single toolkit.

Reply:
For now, we have more stability with pyqt5 than pyside2, if im not wrong. We can always replace the underlying toolkit in the future so for now pyqt5 is fine.